make-it/EmMakefile-AAL

```diff
@@ -32,7 +32,7 @@ $(OBJS)/NoAnalysisLibrary.bc: | no-analysis
 	rm $(OBJS)/NoAnalysis.cpp
 
 no-analysis::
-	$(info Note: Analysis library not found in $(VIREO_SDK_DIR)/private or ../vi.lib/Analysis; omitting from build.)
+	$(info Note: Analysis library not found in $(ANALYSIS_ROOT_DIRECTORY) or Analysis Wrappers not found in $(ANALYSIS_WRAPPERS_SOURCE_DIRECTORY); omitting from build.)
```

Comment:
I would do these changes in a separate commit as this seems to be unrelated. You can still do one pull request for  multiple commits.